lm75bd/lm75bd.c

```diff
@@ -27,7 +27,25 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+
+  uint8_t pointerReg[1] = {0};
```

Comment:
1 element large array is unnecessary. Just use uint8_t set to the pointer value.